packages/twenty-front/src/modules/object-record/hooks/__mocks__/personFragment.ts

```diff
@@ -2,7 +2,11 @@ export const PERSON_FRAGMENT = `
       __typename
       updatedAt
       myCustomObjectId
-      whatsapp
+      whatsapp {
+        primaryPhoneNumber
```

Comment:
nit: could use fragments for composite fields I guess but not a strong requirement!